models/tsodyks_connection_hom.h

```diff
@@ -0,0 +1,331 @@
+/*
+ *  tsodyks_connection_hom.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef TSODYKS_CONNECTION_HOM_H
+#define TSODYKS_CONNECTION_HOM_H
+
+
+/* BeginDocumentation
+  Name: tsodyks_synapse - Synapse type with short term plasticity using homogeneous
```

Comment:
This should be `tsodyks_synapse_hom`,  shouldn't it? The body of the documentation should then also point out which parameters are shared, and contain a link to `tsodyks_synapse`.